### PR TITLE
Use tdex-daemon Buf

### PR DESCRIPTION
**Pull request by Janaka-Steph**

### CODE REVIEW

File: src/api-spec/buf.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 version: v1
 name: buf.build/tdex-network/tdex-daemon
```
Comment (tiero) [inline src/api-spec/buf.yaml:2]: no need for name AFAIK

File: src/api-spec/buf.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+breaking:
```
Comment (tiero) [inline src/api-spec/buf.yaml:2]: you need deps here, pinting to `buf.build/tdex-network/tdex-daemon`

Comment (tiero) [reply src/api-spec/buf.yaml:2]: https://github.com/tdex-network/tdex-daemon/blob/master/api-spec/protobuf/buf.yaml#L3

Comment (Janaka-Steph) [reply src/api-spec/buf.yaml:6]: I need to keep `buf.build/tdex-network/tdex-protobuf` in order to get `types.proto` right?
What about `buf.build/googleapis/googleapis` ?

Comment (tiero) [reply src/api-spec/buf.yaml:6]: no need for google API I guess, used only in trade.proto stuff

**Pull request merged**